### PR TITLE
Add initial specs of Hydra as slurm cluster

### PR DESCRIPTION
**Pull request by wpoely86**

### CODE REVIEW

File: resources.json [hunk 1]
```diff
@@ -0,0 +1,74 @@
+[
+    {
+        "resource": "hydra-ampere_gpu",
+        "name": "VUB Hydra: ampere_gpu partition",
```
Comment (lexming) [inline resources.json]:
```suggestion
        "name": "VUB Hydra: Ampere GPU",
```

File: resources.json [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+        "description": "The ampere_gpu partition of Hydra",
+        "resource_type": "Accelerators machine",
+        "pi_column": "account_name",
+        "shared_jobs": true
+    },
+    {
+        "resource": "hydra-broadwell",
+        "name": "VUB Hydra: broadwell partition",
```
Comment (lexming) [inline resources.json]:
```suggestion
        "name": "VUB Hydra: Broadwell",
```

File: resources.json [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+        "description": "The broadwell partition of Hydra",
+        "resource_type": "Thin",
+        "pi_column": "account_name",
+        "shared_jobs": true
+    },
+    {
+        "resource": "hydra-broadwell_himem",
+        "name": "VUB Hydra: broadwell_himem partition",
```
Comment (lexming) [inline resources.json]:
```suggestion
        "name": "VUB Hydra: Broadwell Hi-Mem",
```

File: resources.json [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+        "description": "The broadwell_himem partition of Hydra",
+        "resource_type": "Shared memory machine",
+        "pi_column": "account_name",
+        "shared_jobs": true
+    },
+    {
+        "resource": "hydra-bvo00025",
+        "name": "VUB Hydra: bvo00025 partition",
```
Comment (lexming) [inline resources.json]:
```suggestion
        "name": "VUB Hydra: bvo00025 GPU",
```

File: resources.json [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+        "description": "The bvo00025 partition of Hydra",
+        "resource_type": "Thin",
+        "pi_column": "account_name",
+        "shared_jobs": true
+    },
+    {
+        "resource": "hydra-ivybridge_mpi",
+        "name": "VUB Hydra: ivybridge_mpi partition",
```
Comment (lexming) [inline resources.json]:
```suggestion
        "name": "VUB Hydra: Ivybridge IB",
```

File: resources.json [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+        "description": "The ivybridge_mpi partition of Hydra",
+        "resource_type": "Thin",
+        "pi_column": "account_name",
+        "shared_jobs": true
+    },
+    {
+        "resource": "hydra-kepler_gpu",
+        "name": "VUB Hydra: kepler_gpu partition",
```
Comment (lexming) [inline resources.json]:
```suggestion
        "name": "VUB Hydra: Kepler GPU",
```

File: resources.json [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+        "description": "The kepler_gpu partition of Hydra",
+        "resource_type": "Accelerators machine",
+        "pi_column": "account_name",
+        "shared_jobs": true
+    },
+    {
+        "resource": "hydra-pascal_gpu",
+        "name": "VUB Hydra: pascal_gpu partition",
```
Comment (lexming) [inline resources.json]:
```suggestion
        "name": "VUB Hydra: Pascal GPU",
```

File: resources.json [hunk 8]
```diff
@@ -0,0 +1,74 @@
...
+        "description": "The pascal_gpu partition of Hydra",
+        "resource_type": "Accelerators machine",
+        "pi_column": "account_name",
+        "shared_jobs": true
+    },
+    {
+        "resource": "hydra-skylake",
+        "name": "VUB Hydra: skylake partition",
```
Comment (lexming) [inline resources.json]:
```suggestion
        "name": "VUB Hydra: Skylake",
```

File: resources.json [hunk 9]
```diff
@@ -0,0 +1,74 @@
...
+        "description": "The skylake partition of Hydra",
+        "resource_type": "Thin",
+        "pi_column": "account_name",
+        "shared_jobs": true
+    },
+    {
+        "resource": "hydra-skylake_mpi",
+        "name": "VUB Hydra: skylake_mpi partition",
```
Comment (lexming) [inline resources.json]:
```suggestion
        "name": "VUB Hydra: Skylake IB",
```

**Review by lexming — CHANGES_REQUESTED** [review]
I would use shorter names that are easier to read on the charts

**Comment by wpoely86** [comment]
@lexming  done

**Review by lexming — APPROVED** [review]
LGTM

**Pull request merged**